spec/features/guider_views_appointments_spec.rb

```diff
@@ -40,6 +57,11 @@ def then_they_see_the_appointment_for_today
     end
   end
 
+  def then_they_see_the_appointments_for_today
+    @page.wait_for_appointments
+    expect(@page).to have_appointments(count: 1)
```

Comment:
I would test that he appointment title is displayed, otherwise we're not sure the "next" button click did anything really.